apps/web/src/app/[locale]/(dashboard)/[wsId]/tasks/boards/utils/taskHelpers.ts

```
@@ -0,0 +1,177 @@
+ * Get status color for task visualization
+ * @param status Task status string
+ * @returns CSS class name for background color
+ */
+export function getStatusColor(status: string): string {
+  switch (status) {
+    case 'done':
+    case 'closed':
+      return 'bg-green-500';
+    case 'active':
+      return 'bg-blue-500';
+    case 'not_started':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)